### PR TITLE
Fix issue #126. Skip locations of the axis when drawing guidelines

### PR DESCRIPTION
**Pull request by vkaramov**

### CODE REVIEW

File: SwiftCharts/Layers/ChartGuideLinesLayer.swift [hunk 1]
```diff
@@ -54,8 +54,18 @@ public class ChartGuideLinesLayerAbstract<T: ChartGuideLinesLayerSettings>: Char
     
     override public func chartViewDrawing(context context: CGContextRef, chart: Chart) {
         let originScreenLoc = self.innerFrame.origin
-        let xScreenLocs = onlyVisibleX ? self.xAxis.visibleAxisValuesScreenLocs : self.xAxis.axisValuesScreenLocs
-        let yScreenLocs = onlyVisibleY ? self.yAxis.visibleAxisValuesScreenLocs : self.yAxis.axisValuesScreenLocs
+        
+        // Skip location of the X axis
+        let xScreenLocs = (onlyVisibleX ? self.xAxis.visibleAxisValuesScreenLocs : self.xAxis.axisValuesScreenLocs).filter{
+            return $0 != originScreenLoc.x
```
Comment (i-schuetz) [inline SwiftCharts/Layers/ChartGuideLinesLayer.swift]: Please write the comments next to the return line, to make clearer what it is addressing.

**Comment by i-schuetz** [comment]
Thanks! I added a comment.

**Comment by vkaramov** [comment]
Should I add more comments?

**Comment by i-schuetz** [comment]
No, I mean only to put the comments next to the filtering line because currently it looks like comment describes the whole thing and that's not the case. Like: 

```swift  

let xScreenLocs = (onlyVisibleX ? self.xAxis.visibleAxisValuesScreenLocs : self.xAxis.axisValuesScreenLocs).filter{
    return $0 != originScreenLoc.x // Skip location of the X axis
}
```

**Comment by i-schuetz** [comment]
Oh, there's something else. It should work also when the axes are at the top or on the right. I think you can use `xAxis` and `yAxis` to get the positions instead of the frame.

**Comment by i-schuetz** [comment]
Was fixed.

**Pull request closed without merging**